test/e2e/specs/default-config/recordedit/add-defaults.spec.js

```diff
@@ -168,7 +177,37 @@ describe('Record Add with defaults', function() {
             expect(foreignKeyInput.getText()).toBe(values.foreign_key_value, "Foreign key input default is incorrect");
             expect(foreignKeyDisabledInput.getText()).toBe(values.foreign_key_disabled_value, "Foreign key disabled default is incorrect");
         });
-
+        //mdhelp page
```

Comment:
The reason I added it in `default-config` was that it does not require any special config for testing. We can certainly refactor mentioned test cases to avoid confusion in future.  
I will create another spec inside `all-features\recordedit` for markdown help page.